app/views/user_mentee_applications/new.html.erb

```diff
@@ -1,33 +1,33 @@
-<h1 class="text-center font-bold text-3xl mb-5">
-  Welcome <%= current_user.first_name %>!
+<h1 class="text-center font-bold text-3xl my-8">
+  Welcome, <%= current_user.first_name %>!
 </h1>
 
 <%= form_with model: @user_mentee_application, local: true, html: { class: 'max-w-prose mx-auto' } do |form| %>
   <div class='flex flex-col ml-5 mr-5'>
     <%= render UserMenteeApplication::SectionComponent.new(header_text: 'Tell us about your journey') do %>
       <div class="flex flex-col mb-5">
-        <%= form.label :learned_to_code, 'How did you learn to code?', class: 'font-bold' %>
```

Comment:
All text on the page was bold, which nullified the efficacy of using bold. Removed font-bold on labels to establish better visual hierarchy.